docs/getting-started-guides/index.md

```diff
@@ -48,7 +48,7 @@ few commands, and have active community support.
 
 - [GCE](/docs/getting-started-guides/gce)
 - [AWS](/docs/getting-started-guides/aws)
-- [Azure](/docs/getting-started-guides/coreos/azure/)
+- [Azure](/docs/getting-started-guides/azure)
```

Comment:
CC: @davidopp 

We had discussed whether we wanted to replace the existing CoreOS/Azure or leave them side-by-side.